Adyen/Utilities/Localization.swift

```diff
@@ -41,7 +42,16 @@ public func localizedString(_ key: LocalizationKey, _ parameters: LocalizationPa
 }
 
 private func fallbackLocalizedString(key: String) -> String {
-    NSLocalizedString(key, tableName: nil, bundle: Bundle.coreInternalResources, comment: "")
+    let localizedFallback = NSLocalizedString(key, tableName: nil, bundle: Bundle.coreInternalResources, comment: "")
+    
+    if localizedFallback != key && localizedFallback.isEmpty == false {
```

Comment:
It can't be equals `key`, right? 🤔

Reply:
Why can add `var nonEmpty: Bool ` extension for String 😆

Reply:
if the key is missing from the `Localizable.strings`, `NSLocalizedString` will return the key. here we need to check for that

Reply:
yeah, actually you're right, here is better.

but can we unit test this in `ADYLocalizationTests`?

Reply:
will do! 👍